### DISCUSSION TITLE
Does Win+Arrow Key work for you?

### DISCUSSION THREAD

**Discussion by kingmustard** [rustdesk/rustdesk#5292] [General]
As of recently, when using those shortcuts within RustDesk, they no longer dock windows to the left, right etc.

The Win key works fine (opening the Start menu), as do the arrow keys.

I am using v1.2.1 on both ends.

**EDIT:** Changing from 'Legacy mode' (which I never changed, so must be the default) to 'Map mode' appears to have fixed it.

**Comment by fufesou** [comment]
@kingmustard Hi, what OSs did you use on both sides? I've tested Win -> Win, and it works as expected.

**Comment by kingmustard** [comment]
> @kingmustard Hi, what OSs did you use on both sides? I've tested Win -> Win, and it works as expected.

Windows 11 on the host, Windows 10 on the remote.

**EDIT:** Changing from 'Legacy mode' (which I never changed, so must be the default) to 'Map mode' appears to have fixed it.
